workshops/forms.py

```diff
@@ -412,3 +413,18 @@ class Meta:
         model = Task
         fields = '__all__'
         widgets = {'person': HiddenInput}
+
+
+class HostForm(forms.ModelForm):
+    domain = forms.CharField(
+        max_length=Host._meta.get_field('domain').max_length,
+        validators=[
+            RegexValidator('[^\w\.-]+', inverse_match=True,
```

Comment:
It matches the inverse, so the regex in `urls.py` is `[\w\.-]+`, here it's `[^\w\.-]+`.